### ISSUE TITLE
Padding area has solid background when background_opacity < 1

### ISSUE THREAD

**Issue by andrei-zz** [kovidgoyal/kitty#7921]
**Describe the bug**
The "padding area" created by the setting `window_padding_width 16 16 16 16` is solid when I set `background_opacity 0.88`

**To Reproduce**
hyprland-git on Arch rolling
Kitty config
```
$ grep -v '^\s*#' ~/.config/kitty/kitty.conf | grep -v '^\s*$'
include ~/.config/kitty/themes/macchiato.conf
font_family family="CommitMono Nerd Font" wght=500
font_size 14.0
cursor_blink_interval 0
scrollback_lines 30000
enable_audio_bell no
window_padding_width 16 16 16 16
background_opacity 0.88
update_check_interval 0
shell_integration no-cursor
```

**Screenshots**
![image](https://github.com/user-attachments/assets/f5f86fd2-1b53-4ba8-a8bd-3c4ad191213b)

**Environment details**
```
kitty 0.36.3 created by Kovid Goyal
Linux archlinux 6.10.10-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 12 Sep 2024 17:21:02 +0000 x86_64
Arch Linux 6.10.10-arch1-1 (/dev/tty)

Running under: Wayland (Hyprland v0.43.0-75-ge20aef7d,) missing: single_pixel_buffer
OpenGL: '4.6 (Core Profile) Mesa 24.2.3-arch1.1' Detected version: 4.6
Frozen: False
Fonts:
  medium: CommitMonoNF-Regular: /usr/share/fonts/OTF/CommitMonoNerdFont-Regular.otf:0
          Features: ()
    bold: CommitMonoNF-Bold: /usr/share/fonts/OTF/CommitMonoNerdFont-Bold.otf:0
          Features: ()
  italic: CommitMonoNF-Italic: /usr/share/fonts/OTF/CommitMonoNerdFont-Italic.otf:0
          Features: ()
      bi: CommitMonoNF-BoldItalic: /usr/share/fonts/OTF/CommitMonoNerdFont-BoldItalic.otf:0
          Features: ()
Paths:
  kitty: /usr/bin/kitty
  base dir: /usr/lib/kitty
  extensions dir: /usr/lib/kitty/kitty
  system shell: /bin/zsh
Loaded config files:
  /home/archuser/.config/kitty/kitty.conf

Config options different from defaults:
background_opacity      0.88
cursor_blink_interval   (0.0, kitty.options.utils.EasingFunction(), kitty.options.utils.EasingFunction())
enable_audio_bell       False
font_family             family="CommitMono Nerd Font" wght=500
font_size               14.0
scrollback_lines        30000
shell_integration       frozenset({'no-cursor'})
window_padding_width    FloatEdges(left=16.0, top=16.0, right=16.0, bottom=16.0)
Colors:
	active_border_color     #b7bdf8   
	active_tab_background   #c6a0f6   
	active_tab_foreground   #181926   
	background              #181818   
	bell_border_color       #eed49f   
	color0                  #494d64   
	color1                  #ed8796   
	color10                 #a6da95   
	color11                 #eed49f   
	color12                 #8aadf4   
	color13                 #f5bde6   
	color14                 #8bd5ca   
	color15                 #a5adcb   
	color2                  #a6da95   
	color3                  #eed49f   
	color4                  #8aadf4   
	color5                  #f5bde6   
	color6                  #8bd5ca   
	color7                  #b8c0e0   
	color8                  #5b6078   
	color9                  #ed8796   
	cursor                  #f4dbd6   
	cursor_text_color       #24273a   
	foreground              #c0c0c0   
	inactive_border_color   #6e738d   
	inactive_tab_background #1e2030   
	inactive_tab_foreground #cad3f5   
	mark1_background        #b7bdf8   
	mark1_foreground        #24273a   
	mark2_background        #c6a0f6   
	mark2_foreground        #24273a   
	mark3_background        #7dc4e4   
	mark3_foreground        #24273a   
	selection_background    #f4dbd6   
	selection_foreground    #24273a   
	tab_bar_background      #181926   
	url_color               #f4dbd6   

Important environment variables seen by the kitty process:
	PATH                                /usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/usr/lib/rustup/bin
	LANG                                en_US.UTF-8
	SHELL                               /bin/zsh
	DISPLAY                             :1
	WAYLAND_DISPLAY                     wayland-1
	USER                                archuser
	XCURSOR_SIZE                        24
	XDG_SESSION_PATH                    /org/freedesktop/DisplayManager/Session1
	XDG_BACKEND                         wayland
	XDG_SEAT                            seat0
	XDG_SESSION_DESKTOP                 Hyprland
	XDG_SESSION_TYPE                    wayland
	XDG_CURRENT_DESKTOP                 Hyprland
	XDG_SEAT_PATH                       /org/freedesktop/DisplayManager/Seat0
	XDG_SESSION_CLASS                   user
	XDG_VTNR                            1
	XDG_SESSION_ID                      2
	XDG_RUNTIME_DIR                     /run/user/1000
	XDG_SCREENSHOTS_DIR                 /home/archuser/Pictures/Screenshots

```

**Additional context**
Can't `kitty --config NONE` for my case.

**Comment by kovidgoyal** [comment]
dup of #7895

**Comment by jnleyva816** [comment]
Just do
`yay -S kitty-git ` 
He fixed it in newest update 0.36.4

**Comment by andrei-zz** [comment]
Sorry for opening a duplicate.
